### PR TITLE
Bump version 0.12.0 & add changelog entries

### PR DESCRIPTION
**Pull request by chrieke**

### CODE REVIEW

File: docs/theme_override_home/main.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
 {% endblock %}
 {% block announce %}
   <style>.md-announce a,.md-announce a:focus,.md-announce a:hover{color:currentColor}.md-announce strong{white-space:nowrap}.md-announce .twitter{margin-left:.2em;color:#00acee}</style>
-    <a href="https://up42.github.io/up42-py/CHANGELOG/"><strong>New version 0.11:</strong> Adds map quicklooks feature & more, see the 📅
+    <a href="https://up42.github.io/up42-py/CHANGELOG/"><strong>New version 0.11.1:</strong> Improves object printouts, adds info & status properties, fixes & docs improvements! See the 📅
```
Comment (rodrigoalmeida94) [inline docs/theme_override_home/main.html]: Maybe we can say "Object representation"? Don't understand the last part, "fixes & docs improvements"

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ pip show up42-py
...
+### [0.11.1](https://pypi.org/project/up42-py/) (2020-10-14)
+
+- Simplify object representation to just relevant information, also simplifies logger messages.
+- Add `.info` property to all objects to get the detialed object information, deprecation process for `get_info`.
+- Add `status` property to job, jobtask and jobcollection objects. Deprecation process for `get_status`.
+- Add selection of job mode for `.get_jobs`.
+- Add description of initialization of each object to code reference.
+- Move sdk documentation to custom subdomain "sdk.up42.com".
+- Unpin geopandas version.
+- Simplify mock tests & put http mocks in object fixtures to simplify future tests.
```
Comment (rodrigoalmeida94) [inline CHANGELOG.md:28]: Is this included in the package? If not we should remove from changelog

**Review by rodrigoalmeida94 — CHANGES_REQUESTED** [review]
Just some small corrections

File: up42/_version.txt [hunk 3]
```diff
@@ -1 +1 @@
-0.11.0
\ No newline at end of file
+0.11.1
```
Comment (markusuwe) [inline up42/_version.txt:1]: This is more than a bug fix, therefore need to bump to 0.12.0
```suggestion
0.12.0
```

**Review by markusuwe — APPROVED** [review]
LGTM

File: docs/theme_override_home/main.html (hunk 1 above)
Comment (chrieke) [reply docs/theme_override_home/main.html]: yup, edited & removed last part.

File: CHANGELOG.md (hunk 2 above)
Comment (chrieke) [reply CHANGELOG.md:28]: clarified

File: up42/_version.txt (hunk 3 above)
Comment (chrieke) [reply up42/_version.txt]: thanks

**Review by rodrigoalmeida94 — APPROVED** [review]

**Pull request merged**